### PR TITLE
Support additional jar-libraries

### PR DESCRIPTION
**Pull request by AlexandrPrendota**

### CODE REVIEW

File: versions/1.0.7/build.gradle [hunk 1]
```diff
@@ -19,15 +20,49 @@ dependencies {
...
+
     compile fileTree(dir: projectDir.toString() + File.separator + "kotlin", include: '*.jar')
 }
 
 task copyKotlinLibs(type: Copy) {
     from configurations.kotlinJVM
     into projectDir.toString() + File.separator + "kotlin"
 }
+
+task copyLibs(type: Copy) {
```
Comment (satamas) [inline versions/1.0.7/build.gradle]: We don't need this task. We have to copy libraries only for Idea completion/highlighting/e.t.c. Idea don't need this libraries, so we can copy libs to war directly

File: versions/1.1.60/src/main/java/org/jetbrains/webdemo/kotlin/impl/KotlinWrapperImpl.java [hunk 2]
```diff
@@ -49,8 +53,9 @@ public void init(List<Path> javaLibraries, KotlinVersionConfig config) {
         WrapperLogger.init(kotlinVersion);
         wrapperFolder = KotlinWrappersManager.INSTANCE.getWrappersDir().resolve(kotlinVersion);
         jarsFolder = wrapperFolder.resolve("kotlin");
```
Comment (satamas) [inline versions/1.1.60/src/main/java/org/jetbrains/webdemo/kotlin/impl/KotlinWrapperImpl.java]: Why we need this?

File: versions/1.1.60/src/main/java/org/jetbrains/webdemo/kotlin/impl/KotlinWrapperImpl.java [hunk 3]
```diff
@@ -49,8 +53,9 @@ public void init(List<Path> javaLibraries, KotlinVersionConfig config) {
...
+        userLibraries = wrapperFolder.resolve("libraries");
```
Comment (satamas) [inline versions/1.1.60/src/main/java/org/jetbrains/webdemo/kotlin/impl/KotlinWrapperImpl.java]: UserLibraries isn't correct name for this field.

File: versions/1.2.21/src/main/java/org/jetbrains/webdemo/kotlin/impl/KotlinWrapperImpl.kt [hunk 4]
```diff
@@ -90,12 +82,16 @@ class KotlinWrapperImpl : KotlinWrapper {
...
+     *
+     * @return list of [Path] to library
+     */
+    override fun getKotlinLibraries(): MutableList<Path> {
         val libraries = ArrayList<Path>()
-        libraries.add(jarsFolder!!.resolve("kotlin-runtime-$kotlinBuild.jar"))
-        libraries.add(jarsFolder!!.resolve("kotlin-reflect-$kotlinBuild.jar"))
-        libraries.add(jarsFolder!!.resolve("kotlin-test-$kotlinBuild.jar"))
-        libraries.addAll(compileTimeLibraries)
+        userLibFolder!!.toFile().listFiles(JarLibraryFileFilter())?.forEach { libraries.add(it.toPath()) }
```
Comment (satamas) [inline versions/1.2.21/src/main/java/org/jetbrains/webdemo/kotlin/impl/KotlinWrapperImpl.kt]: Use lambda here. And looks like !! is used extremely often.

File: versions/1.0.7/build.gradle (hunk 1 above)
Comment (AlexandrPrendota) [reply versions/1.0.7/build.gradle]: I created this task just for testing

**Pull request merged**